modules/Campaigns/WebToPersonCapture.php

```diff
@@ -306,8 +310,16 @@
             if (isset($mod_strings['LBL_THANKS_FOR_SUBMITTING'])) {
                 echo $mod_strings['LBL_THANKS_FOR_SUBMITTING'];
             } else {
-                //If the custom module does not have a LBL_THANKS_FOR_SUBMITTING label, default to this general one
-                echo 'Success';
+                
+                if(isset($errors) && $errors) {
+                    echo 'Success but some error occured: <br>'; // TODO: translate strings and template!
```

Comment:
I don't think we should be alert the user who is submitting the form to say there is an error.
Can we move this to logs?